node/impl/full.go

```diff
@@ -37,4 +40,40 @@ func (n *FullNodeAPI) CreateBackup(ctx context.Context, fpath string) error {
 	return backup(n.DS, fpath)
 }
 
+func (n *FullNodeAPI) NodeStatus(ctx context.Context) (status api.NodeStatus, err error) {
+	curTs, err := n.ChainHead(ctx)
+	if err != nil {
+		return status, err
+	}
+
+	status.SyncStatus.Epoch = uint64(curTs.Height())
+	timestamp := time.Unix(int64(curTs.MinTimestamp()), 0)
+	delta := time.Since(timestamp).Seconds()
+	status.SyncStatus.Behind = uint64(delta / 30)
+
+	pis, err := n.NetPeers(ctx)
+	if err != nil {
+		return status, err
+	}
+
+	scores, err := n.NetPubsubScores(ctx)
+	if err != nil {
+		return status, err
+	}
+
+	scoreMap := make(map[peer.ID]float64)
+	for _, score := range scores {
+		scoreMap[score.ID] = score.Score.Score
+	}
+
+	for _, pi := range pis {
+		score := scoreMap[pi.ID]
+		if score > -1000 {
```

Comment:
Where is this threshold coming from?

Reply:
our score configuration, no good way to reference it.

Reply:
Can we create a constant in right place and reference it from here? I'm afraid that if we change the threshold at origin, we will forget to update it here.

Reply:
probably.

Reply:
done.